src/content/web3/index.md

```diff
@@ -0,0 +1,148 @@
+---
+title: What is web3 and why is it important?
+description: An introduction to web3 - the next evolution of the internet - and why it matters.
+lang: en
+sidebar: true
+---
+
+# Introduction to web3 {#introduction}
+
+Centralization has helped onboard billions of people to the internet and created the stable, robust infrastructure on which it lives. At the same time, a handful of centralized entities have a stronghold on large swathes of the internet, unilaterally deciding what should and should not be allowed.
+
+Web3 is the answer to this dilemma. Instead of an internet monopolized by large technology companies, web3 embraces decentralization and is being built, operated, and owned by its users. Web3 will put power in the hands of individuals rather than corporations.
+Before we talk about web3, let's explore how we got here.
+
+<Divider />
+
+## The early internet {#early-internet}
+
+Most people think of the internet as a continuous pillar of modern life—it was invented and has just existed since. However, the internet most of us know today is quite different from originally imagined. To understand this better, it's helpful to break the internet's short history into loose periods—web 1.0 and web 2.0.
+
+### Web1: Read-Only {#web1}
```

Comment:
Is there a reason we go between `WebX` syntax and `Web X.0` syntax? Just curious I guess, wondering if it's needed
We also should standardize between `WebX` and `webX` in the middle of a sentence. Is there any such consensus? I feel like I lean towards lowercase, but not a hill I care to die on in the slightest

Reply:
> Is there a reason we go between WebX syntax and Web X.0 syntax? Just curious I guess, wondering if it's needed

It's kinda confusing because Web 1.0 and Web 2.0 are pretty standard. Gavin coined it as 'Web 3.0' and somewhere along the line it became just 'Web3'.

So the currently used terms seem to be:

- Web 1.0
- Web 2.0
- Web3

Kinda messy... 

I think the best solution is to exclusively use Web 1.0, and Web 2.0. Then, use Web3.0 for the read-write-own heading and introductory paragraph and prepend an explanation that it evolved to become known as Web3

Reply:
> We also should standardize between WebX and webX in the middle of a sentence. Is there any such consensus? I feel like I lean towards lowercase, but not a hill I care to die on in the slightest

Yup, agree. Also no strong opinions. I'll do a little digging into what the current standard leans towards.

Reply:
> I think the best solution is to exclusively use Web 1.0, and Web 2.0. Then, use Web3.0 for the read-write-own heading and introductory paragraph and prepend an explanation that it evolved to become known as Web3

If the first two were called `Web 1.0` and `Web 2.0`, it feels like we should call it `Web 3.0` everywhere.  If people want to shorthand that elsewhere that is up to them, but I feel like for more formal documentation, FAQs, websites, etc. we should use the more consistent/strict terminology.

Reply:
Hmm, I don't think I'd agree here.

I don't think going against the common nomenclature for consistency of style makes sense. The term has evolved (e.g. Gavin, despite being credited as coining 'Web 3.0' has the 'Web3 Foundation')  and I think we should refer to it as it known now. Otherwise, we might leave readers confused (i.e. they've read about Web 3.0, is Web3 different?)

Reply:
'> > I think the best solution is to exclusively use Web 1.0, and Web 2.0. Then, use Web3.0 for the read-write-own heading and introductory paragraph and prepend an explanation that it evolved to become known as Web3
> 
> If the first two were called `Web 1.0` and `Web 2.0`, it feels like we should call it `Web 3.0` everywhere. If people want to shorthand that elsewhere that is up to them, but I feel like for more formal documentation, FAQs, websites, etc. we should use the more consistent/strict terminology.'

Web3 is not shorthand for Web 3.0. There is a missing gap here. 

There is:

Web 1.0 - Read-only.
Web 2.0 - Read-write.
Web 3.0 - Semantic Web. 

The final one did exist in some textbooks (some I used to teach myself), but never stuck in the mainstream. 

Then there is our version, Web3 or web3. This is how the term is used and understood by the majority of Ethereum users. They do not think it is shorthand for Web 3.0 (and, technically speaking, it's not given that means the Semantic Web) and I believe they would consider us idiosyncratic to be using the more outdated 3.0. 

It's a confusing term, but I think over time we'll see Web3/web3 stabilise nicely.

Reply:
On a different note, some sections use web3 and some Web3. We should pick one.